src/pytest_plugins/execute/eth_config/execute_eth_config.py

```diff
@@ -171,3 +178,64 @@ def test_eth_config_last_fork_id(
                 f"{received_fork_id} != "
                 f"{expected_last_fork_id}"
             )
+
+
+def test_eth_config_majority(
+    all_rpc_endpoints: Dict[str, List[EthRPC]],
+) -> None:
+    """Queries devnet exec clients for their eth_config and fails if not all have the same response."""  # noqa: E501
+    responses = dict()  # Dict[exec_client_name : response] # noqa: C408
+    client_to_url_used_dict = dict()  # noqa: C408
+    for exec_client in all_rpc_endpoints.keys():
+        # try only as many consensus+exec client combinations until you receive a response
+        # if all combinations for a given exec client fail we panic
+        for eth_rpc_target in all_rpc_endpoints[exec_client]:
+            response = eth_rpc_target.config(timeout=10)
+            if response is None:
+                # safely split url to not leak rpc_endpoint in logs
+                logger.warning(
+                    f"When trying to get eth_config from {eth_rpc_target} a problem occurred"  # problem itself is logged by .config() call # noqa: E501
+                )
+                continue
+
+            response_str = json.dumps(response.model_dump(mode="json"))
+            responses[exec_client] = response_str
+            client_to_url_used_dict[exec_client] = (
+                eth_rpc_target.url
+            )  # remember which cl+el combination was used  # noqa: E501
+            logger.info(f"Response of {exec_client}: {response_str}\n\n")
+
+            break  # no need to gather more responses for this client
+
+    assert len(responses.keys()) == len(all_rpc_endpoints.keys()), (
+        "Failed to get an eth_config response "
+        f" from each specified execution client. Full list of execution clients is "
+        f"{all_rpc_endpoints.keys()} but we were only able to gather eth_config responses "
+        f"from: {responses.keys()}\n"
+        "Will try again with a different consensus-execution client combination for "
+        "this execution client"
+    )
+    # determine hashes of client responses
+    client_to_hash_dict = dict()  # Dict[exec_client : response hash] # noqa: C408
+    for client in responses.keys():
+        response_bytes = json.dumps(responses[client], sort_keys=True).encode("utf-8")
```

Comment:
`responses[client]` is already the dumped json object as a string, so no need to re-dump it here.
```suggestion
        response_bytes = responses[client].encode("utf-8")
```
Test was failing for me locally in nethermind vs besu because the `sort_keys` had no affect on the string, but this fixes it.